### ISSUE TITLE
Seeking help

### ISSUE THREAD

**Issue by Yanruoxi001** [Bunsly/HomeHarvest#81]
May I ask if many cities are unable to obtain data under the "Sold" tag when crawling house information. The error message is as follows. What is the reason for this?

'NoneType' object has no attribute 'value'

**Comment by Yanruoxi001** [comment]
Yes, I am using a website. Thank you very much for your answer. I will try other methods.

**Comment by ZacharyHampton** [comment]
I will upgrade our website's homeharvest version to the latest.

**Comment by Yanruoxi001** [comment]
Thank you very much!

**Comment by ZacharyHampton** [comment]
Upgraded, let me know if you have any further issues.

**Comment by Yanruoxi001** [comment]
Thank you very much for your work. I have a few questions to consult with you.
(1) Is it the default way to crawl data for all dates when the "last day" and "radius" columns in the web version are empty? As shown in Figure 1, I can crawl 1214 pieces of data from Alaska;
(2) When setting "Listing type: Sold" in most cities, data cannot be crawled when the "past day" and "radius" columns are empty. At this point, it is necessary to set "past day" to proceed. But in a few cities, data can also be crawled when the "last day" and "radius" columns are empty. The error reminder shown in Figure 2.
![02](https://github.com/Bunsly/HomeHarvest/assets/152482574/a9ea6de3-d6cf-4097-8c3a-e7b04f654df3)
![01](https://github.com/Bunsly/HomeHarvest/assets/152482574/5c1c73e4-288e-4992-a266-2abbb5dfad8b)

**Comment by ZacharyHampton** [comment]
I fixed the above error in the latest version push (v0.3.30).
